src/github.com/getlantern/connpool/connpool.go

```diff
@@ -9,9 +9,7 @@ import (
 )
 
 const (
-	DefaultClaimTimeout         = 10 * time.Minute
-	DefaultRedialDelayIncrement = 50 * time.Millisecond
-	DefaultMaxRedialDelay       = 1 * time.Second
+	DefaultClaimTimeout = 10 * time.Minute
```

Comment:
+1 on making these constants rather than using magic numbers.  -1 on exporting names unless we think external libraries should be able to use them.